Packs/VMRay/pack_metadata.json

```
@@ -0,0 +1,25 @@
+{
+    "name": "VMRay",
+    "description": "Ransomware analysis sandboxing.",
+    "support": "xsoar",
+    "serverMinVersion": "4.0.0",
```

please remove

---

```suggestion
    "currentVersion": "1.0.0",
```
@liorblob I think you removed "currentVersion": "1.0.0".
apply this to correct.